frontend/app/src/components/common/IconButton.tsx

```diff
@@ -8,7 +8,7 @@ interface BProps {
   background: string;
   width: string;
 }
-const B = styled(EuiButton)<BProps>`
+const B = styled(EuiButton) <BProps>`
```

Comment:
do you know the reason why you're adding these changes, our prettier script will just add them back and it makes it more difficult to review